### DISCUSSION TITLE
A flet-based installer for alternative Android distributions.

### DISCUSSION THREAD

**Discussion by tsterbak** [flet-dev/flet#541] [Show and tell]
Hi all :)
I'm currently working on a project to make installing alternative Android distributions (like LineageOS) nice and easy.
Thanks to the awesome flet, I was able to get to a working, graphical tool quite fast. The code still has some (or a lot) of rough edges, but hopefully they will be smoothed out over time.

![Screenshot](https://forum.xda-developers.com/attachments/screenshot-openandroidinstaller-png.5743119/)

Check it out and feel free to comment :)

https://github.com/openandroidinstaller-dev/openandroidinstaller

**Comment by ndonkoHenri** [comment]
Is it already functional, or still in dev..?

**Reply by tsterbak** [reply]
The tool works for some devices and a workflow to add more devices is already implemented :)
If you try it, I'm happy about all kinds of feedback! :)

**Reply by ndonkoHenri** [reply]
Toll.
Wie findest du Flet? 
_Echt cool_ oder?

**Comment by clstaudt** [comment]
Great to see more real-world applications written with flet. While I couldn't run the Android installation process yet, the UI works fine on macOS.

![Screenshot 2022-11-04 at 16 56 19](https://user-images.githubusercontent.com/875194/200020420-a99b7397-3ec2-4da2-8c0d-3d38bd0bf743.png)
